skyportal/handlers/api/stream.py

```diff
@@ -0,0 +1,168 @@
+from marshmallow.exceptions import ValidationError
+
+from baselayer.app.access import auth_or_token, permissions
+from ..base import BaseHandler
+from ...models import (
+    DBSession,
+    Stream,
+)
+
+
+class StreamHandler(BaseHandler):
+    @auth_or_token
```

Comment:
What are the use cases for exposing this endpoint to a general user?

Reply:
Couldn't think of anything specific, but thought there might be, and there isn't much criminal in getting this information. But ok, changed for `@permissions(["System admin"])`.